pandas/tests/test_resample.py

```diff
@@ -1173,27 +1173,19 @@ def test_resample_frame_basic(self):
         df.resample('M', kind='period').mean()
         df.resample('W-WED', kind='period').mean()
 
-    def test_resample_loffset(self):
+    @pytest.mark.parametrize('loffset', [timedelta(minutes=1),
+                                         '1min', Minute(1),
+                                         np.timedelta64(1, 'm')])
+    def test_resample_loffset(self, loffset):
```

Comment:
Can we put a comment referencing the issue next to the relevant input value to `loffset`?